content/v2.0/reference/cli/influx/config/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: influx config
+description: The 'influx config' command and subcommands manage multiple InfluxDB connection configurations.
+menu:
+  v2_0_ref:
+    name: influx config
+    parent: influx
+weight: 101
+v2.0/tags: [config]
+---
+
+The `influx config` command manages multiple InfluxDB connection configurations file (by default, stored at `~/.influxdbv2/configs`). Each connection includes a URL, token, associated organization, and active setting. InfluxDB reads the token from the active connection configuration, so you don't have to manually enter a token to log into InfluxDB.
+
+## Usage
+```
+influx config [flags]
+influx config [command]
+```
+
+## Subcommands
+| Subcommand                                         | Description   |
+|:----                                               |:-----------   |
+| [create](/v2.0/reference/cli/influx/config/create) | Create a new connection configuration to include in the configuration (`config`) file (by default, at `~/.influxdbv2/config`).
```

Comment:
This is just a short summary table, so I'd keep these descriptions brief. "Create a new connection configuration" should be fine here. The other content is just details that can be explained elsewhere.